### DISCUSSION TITLE
Inconsistent emoji sizes and use of g-emoji in issues

### DISCUSSION THREAD

**Discussion by raihle** [community/community#61946] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Emoji are not rendered at a consistent size in issues - even within the same issue.

<details>
<summary>Here's a screenshot of an issue with emoji used at various text sizes (annotated with the intended size):</summary>

![image](https://github.com/community/community/assets/985768/f13268cd-5c09-46a4-b91a-dc98f49cca26)

</details>

Looking at the DOM, the "big" emoji are all wrapped in `<g-emoji>` tags, while the small ones are "normal" text. The rendering seems consistent between page views, but changes after editing the post - for example by checking a checkbox in a task list.

<details>
<summary>Here's a screenshot of the same issue in edit mode:</summary>

![image](https://github.com/community/community/assets/985768/db74cca5-e433-4b1e-a212-467623ebcc2c)

</details>

I mainly use Firefox (115.0.2) on macOS (13.4.1), but the issue appears in Chrome and likely other browsers as well.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by birros** [comment]
We are encountering a similar issue on [media-kit](https://github.com/media-kit/media-kit).
This results in inconsistent rendering of checkmarks in the `README`.

<details>
<summary>Screenshot</summary>
<img width="149" alt="g-emoji-font-size-bug" src="https://github.com/community/community/assets/32417752/aab08d8c-161c-4563-8166-dc60916af54c">
</details>

This is caused by certain emojis being replaced by a `g-emoji` with a different `font-size` than the original text ([global-9f6ffd386e47.css](https://github.githubassets.com/assets/global-9f6ffd386e47.css)):
```scss
@media (-webkit-min-device-pixel-ratio: 2) and (-webkit-min-device-pixel-ratio: 0), (-webkit-min-device-pixel-ratio: 2) and (min-resolution: 0.001dpcm)
g-emoji {
    font-size: 1.25em;
}
```

**Comment by vladimirvolek** [comment]
:(
<img width="838" alt="image" src="https://github.com/community/community/assets/3112191/76985086-850d-4c90-b0b7-092885872fc8">

<img width="850" alt="image" src="https://github.com/community/community/assets/3112191/47f3c618-3b62-4a51-82a4-c1c54670b224">

**Comment by vladimirvolek** [comment]
wow Is it fixed?

![image](https://github.com/community/community/assets/3112191/9a33fe8c-73d1-44c2-bd28-e5dc0b671497)
